conf/routes

```diff
@@ -4,8 +4,8 @@
 # ~~~~
 
 # An example controller showing a sample home page
-GET        /                    controllers.HomeController.index
-GET        /devices             controllers.DevicesController.getDevices
+GET        /                    controllers.DevicesController.getDevicesHtml()
+GET        /devices.json        controllers.DevicesController.getDevicesJson()
```

Comment:
I changed this because I was going to call the page `devices` but then I changed my mind and it went to `/`. Do you like more like this or like it was?